wurst/objects/abilities/beastmaster/Druid/BarkSkin.wurst

```diff
@@ -0,0 +1,89 @@
+package BarkSkin
+
+// Standard Library Imports:
+import Icons
+import ChannelAbilityPreset
+import ClosureEvents
+import ClosureTimers
+import BuffObjEditing
+import BuffIds
+import Abilities
+import DamageEvent
+
+// Local Imports:
+import LocalObjectIDs
+import ToolTipsUtils
+
+
+let COOLDOWN = 45.
+let MANA_COST = 15
+let ARMOR_BONUS = 4
+let THORNS_DMG = .15
+let DURATION = 15.
+let CAST_RANGE = 15.
+let BUTTON_POS_X = 3
+let BUTTON_POS_Y = 2
+let HOTKEY = "R"
+let NAME = "Bark Skin"
+let ICON = Icons.bTNThornShield
+let TOOLTIP_BASIC = "Bark Skin"
+let TOOLTIP_EXT   = "Imbue your target with the strength of the forest, giving it " + ARMOR_BONUS.toReal().toToolTipGreen()
+    + " armor for " + DURATION.toToolTipOrange() + " seconds."
+    + makeToolTipCooldown(COOLDOWN)
+let BUFF_TOOLTIP = "The spirits of nature are protecting this unit and reflecting damage it takes back."
+let BUFF_ID = compiletime(BUFF_ID_GEN.next())
+
+@compiletime function createBarkSkinBuff() returns BuffDefinition
+    return new BuffDefinition(BUFF_ID, BuffIds.innerFire)
+        ..setIcon(ICON)
+        ..setName(1, NAME)
+        ..setArtTarget(1, Abilities.spikeBarrier)
```

Comment:
@HallerJohn mentioned that he didn't find the spiked carapace ability effect, here's the effect, it was named "ThornyShield"

I ran some tests, I believe following setting are decent :
```
        ..setTargetAttachments(1, 2)
        ..setArtTarget(1, commaList(
            Abilities.thornyShieldTargetChestMountLeft,
            Abilities.thornyShieldTargetChestMountRight
            ))
        ..setTargetAttachmentPoint0(1, "chest")
        ..setTargetAttachmentPoint1(1, "chest")
```